plugins/actionlog/joomla/services/provider.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * @package     Joomla.Plugin
+ * @subpackage  System.Actionlog
+ *
+ * @copyright   (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+use Joomla\CMS\Extension\PluginInterface;
+use Joomla\CMS\Plugin\PluginHelper;
+use Joomla\Database\DatabaseInterface;
+use Joomla\DI\Container;
+use Joomla\DI\ServiceProviderInterface;
+use Joomla\Event\DispatcherInterface;
+use Joomla\Plugin\Actionlog\Joomla\Joomla;
+
+return new class implements ServiceProviderInterface {
+	/**
+	 * Registers the service provider with a DI container.
+	 *
+	 * @param   Container  $container  The DI container.
+	 *
+	 * @return  void
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function register(Container $container)
+	{
+		$container->set(
+			PluginInterface::class,
+			function (Container $container)
+			{
+				$plugin = new Joomla(
+					$container->get(DispatcherInterface::class),
+					(array) PluginHelper::getPlugin('system', 'cache'),
```

Comment:
```suggestion
					(array) PluginHelper::getPlugin('system', 'cache')
```
Its function parameters and not an array, so there should not be a comma after the last element like it would be if it was an array.